CSharpException/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using CSharpException;
+
+
+Console.WriteLine("Please enter side 1 length");
+try {
+    double side1 = Convert.ToDouble(Console.ReadLine());
+    if (side1 == 0)
```

Comment:
you have already thrown the exception in Triangle class. So no need to throw in try construction. Here you need to enter numbers into console and pass them to object of Triangle class.

Reply:
it's made for not entering 3 wrong numbers one by now but exit the program after any wrong value

Reply:
You can make up another way to check this case. Introduce a new bool variable(isNumberNull) and put try catch construct in a loop (while(!isNumberNull)). And if the user enter null makes this variable true and add "continue;" in if construction to return to condition in while loop to enter number again.

Reply:
same comment :)